### DISCUSSION TITLE
having trouble building latest drop

### DISCUSSION THREAD

**Discussion by myusrn** [LSPosed/MagiskOnWSALocal#731] [Q&A] [answered]
i'm looking to build the latest drop, hopefully resulting in a 2309.40000.8.0 install as discussed in https://blogs.windows.com/windows-insider/2023/10/19/update-to-windows-subsystem-for-android-on-windows-11-october-2023/.

when i clone current https://github.com/LSPosed/MagiskOnWSALocal.git and execute ./scripts/run.sh [ or ./scripts/install_deps.sh followed by ./scripts/build.sh ] it produces the following errors. any insights as to how i can get around this?
  
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
Add device administration features
sed: can't read /cts/a \    &lt;feature name="android.software.device_admin" /&gt: No such file or directory
done
. 
. .
. . .
Create system images
mk_image_umount: /tmp/wsa-build-ld50hy7yZf_/system_root_merged/vendor /tmp/wsa-build-ld50hy7yZf_/wsa/x64/vendor.img /tmp/wsa-build-ld50hy7yZf_/upper/vendor ext4
ERROR: Not yet implemented
Build: an error has occurred, exit

Cleanup Work Directory
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

**Comment by sn-o-w** [comment, marked as answer]
https://github.com/LSPosed/MagiskOnWSALocal/issues/717#issuecomment-1779576833
